### PR TITLE
Wp 525 enable video for webrtc

### PR DESCRIPTION
**Pull request by pc-m**

### CODE REVIEW

File: contribs/examples/enabled-video.sh [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+# Parse command line arguments
+while getopts t:u: flag
+do
+    case "${flag}" in
+        t) TENANT_UUID="${OPTARG}";;
+        u) AUTH_USERNAME="${OPTARG}";;
+    esac
+done
+
+if [[ ! -v TENANT_UUID ]]; then
```
Comment (sduthil) [inline contribs/examples/enabled-video.sh]: This condition is also true if `$TENANT_UUID=''`. So I think `[ -z "$TENANT_UUID" ]` would be better.

File: contribs/examples/enabled-video.sh [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+set -u  # fail if variable is undefined
+set -o pipefail  # fail if command before pipe fails
+
+usage() {
+    echo "Usage: $0 -t <tenant-uuid> [-u <auth-username>]"
+    exit 1
+}
+
+# Check for dependencies
+if ! command -v wazo-auth-cli &> /dev/null; then
```
Comment (sduthil) [inline contribs/examples/enabled-video.sh]: Nice! I did not know about the `command -v` trick! Thank you!

File: contribs/examples/enabled-video.sh [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    esac
+done
+
+if [[ ! -v TENANT_UUID ]]; then
+    usage
+fi
+
+# Get a TOKEN
+if [[ -v AUTH_USERNAME ]]; then
+    echo -n "Password: "
```
Comment (sduthil) [inline contribs/examples/enabled-video.sh]:
```suggestion
    echo -n "Password for $AUTH_USERNAME: "
```

File: contribs/examples/enabled-video.sh [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+    exit 3
+fi
+
+if [[ ! -v WEBRTC_VIDEO_TEMPLATE_UUID ]]; then
+    echo 'Missing "webrtc_video" template cannot continue'
+    exit 3
+fi
+
+# Fetch all WebRTC video endpoints
+readarray -t WEBRTC_VIDEO_ENDPOINT_UUIDS < <(wazo-confd-cli --token ${TOKEN} endpoint sip list --template ${WEBRTC_VIDEO_TEMPLATE_UUID} --tenant ${TENANT_UUID} -f value | awk -F ' ' '{ print $1 }')
```
Comment (sduthil) [inline contribs/examples/enabled-video.sh]:
```suggestion
readarray -t WEBRTC_VIDEO_ENDPOINT_UUIDS < <(wazo-confd-cli --token ${TOKEN} endpoint sip list --template ${WEBRTC_VIDEO_TEMPLATE_UUID} --tenant ${TENANT_UUID} -f value -c uuid')
```

File: contribs/examples/enabled-video.sh [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+# Find WebRTC endpoints that do not have Video enabled
+echo -n 'Updating WebRTC endpoints'
+for endpoint_uuid in $(wazo-confd-cli --token ${TOKEN} endpoint sip list --template ${WEBRTC_TEMPLATE_UUID} --tenant ${TENANT_UUID} -f value | awk -F ' ' '{ print $1 }'); do
```
Comment (sduthil) [inline contribs/examples/enabled-video.sh]:
```suggestion
for endpoint_uuid in $(wazo-confd-cli --token ${TOKEN} endpoint sip list --template ${WEBRTC_TEMPLATE_UUID} --tenant ${TENANT_UUID} -f value -c uuid); do
```

**Review by sduthil — CHANGES_REQUESTED** [review]

File: contribs/examples/enabled-video.sh (hunk 4 above)
Comment (pc-m) [reply contribs/examples/enabled-video.sh]: very nice, thanks for the info

**Review by sduthil — APPROVED** [review]

**Pull request merged**